### PR TITLE
Title of Roller shutter position, Position value after adapter start, Visibility of power meters

### PR DESCRIPTION
**Pull request by holger-dunst**
- After a user comment I changed the title of the roller shutter position to "Opening level in %"
- To show the current value of the roller shutter position, I immediately set the value after creating the property. I only implemented it for the position, because I don't own other devices and can't test the change.
- I set the power meter properties to invisible if there is a main power meter. What do you think?

### CODE REVIEW

**Comment by tim-hellhake** [comment]
Why do you want to hide the individual power meters?

**Comment by holger-dunst** [comment]
One user wrote during the last PR: "I see 3 Power statements: Power 0, Power 1, Power all. Even though I know where the three levels come from, only "Power all" is relevant for the roller shutter mode (in contrast to relay mode)".

That's why I implemented it this way.
But I'm not really sure. 
At first I thought if I set a property to invisible, it would still be usable in the rules manager. Unfortunately it is not. 

The displayed power values are also confusing. For me a value is only displayed when the roller shutter has finished its action. Then it goes directly back to 0, so not really useful.
You could use the properties to determine the direction. Powermeter 0 -> up, Powermeter 1 -> down, but I do not need this feature.

I'm not sure what makes sense here.

File: src/shelly-device.ts [hunk 1]
```diff
@@ -59,11 +59,12 @@ export class ShellyDevice extends Device {
     }
 
     private configureRollerPosition(): void {
-        new RollerPositionProperty(this, 'rollerPosition', 'Relative roller shutter position', (value) => {
+        const rollerPositionProperty = new RollerPositionProperty(this, 'rollerPosition', 'Opening level in %', (value) => {
             this.device.setRollerPosition(value).catch(reason => {
                 console.log(reason);
             })
         });
+        rollerPositionProperty.setValue(this.device.rollerPosition);
```
Comment (tim-hellhake) [inline src/shelly-device.ts:67]: I think this needs to be `setCachedValueAndNotify`.
The method `setValue` will also send the value to the shelly.
```suggestion
        rollerPositionProperty.setCachedValueAndNotify(this.device.rollerPosition);
```

**Comment by tim-hellhake** [comment]
> only "Power all" is relevant for the roller shutter mode (in contrast to relay mode)"

I see.
That makes perfect sense.
But I think you deactivated it for both modes.
I already have an idea of how to do that.
Would it be ok for you if I solve it directly on my master branch?

**Comment by tim-hellhake** [comment]
> only "Power all" is relevant for the roller shutter mode (in contrast to relay mode)"

I see.
That makes perfect sense.
But I think you deactivated it for both modes.
That's what confused me.

> At first I thought if I set a property to invisible, it would still be usable in the rules manager. Unfortunately it is not.

There is currently no way to make things invisible in the gateway.
If you set `visible` to `false` the property is simply filtered out.

> Then it goes directly back to 0, so not really useful.

Yeah, that's true.
I think the best way would be to deactivate power values completely in roller mode.

**Comment by holger-dunst** [comment]
Thanks, Tim, for your suggestions.
I reverted the property hiding change and I am using now the method setCachedValueAndNotify.

**Pull request merged**